config.yaml

```diff
@@ -0,0 +1,42 @@
+---
+options:
```

Comment:
We may want to be lazy and only support a limited amount of options that we are confident work (ex. drop ipv6 until people ask for it)

Reply:
I do not think we should drop IPv6, we are in 2022, and we already have the support for it. Multiple carriers already support IPv6 on their 5G deployments, so the request will come pretty fast anyway.

Reply:
I added a warning to the `README`.